[2018-10-03 20:26:59] [    INFO] --- Setup logging. Log file: output_v2/s1_p_2_2_4/CIFAR10/model_name=CNN_s1_2_2_4/optimizer_name=Adam/no_cuda=True/03-10-18-20h-26m-59s/logs.txt (setup.py:511)
[2018-10-03 20:26:59] [    INFO] --- Arguments saved to: output_v2/s1_p_2_2_4/CIFAR10/model_name=CNN_s1_2_2_4/optimizer_name=Adam/no_cuda=True/03-10-18-20h-26m-59s/args.txt (setup.py:514)
[2018-10-03 20:26:59] [    INFO] --- Initialize Tensorboard SummaryWriter (setup.py:519)
[2018-10-03 20:27:01] [ WARNING] --- Git status is ignored! (RunMe.py:205)
[2018-10-03 20:27:01] [    INFO] --- Randomly chosen seed is: 1823005204 (setup.py:596)
[2018-10-03 20:27:01] [    INFO] --- Model CNN_s1_2_2_4 expects input size of (32, 32) (image_classification.py:60)
[2018-10-03 20:27:01] [    INFO] --- Loading CIFAR10 from:datasets/CIFAR10 (setup.py:233)
[2018-10-03 20:27:01] [    INFO] --- Verifying the dataset integrity - quick (dataset_integrity.py:182)
[2018-10-03 20:27:02] [    INFO] --- Newly measured timestamp: Tue Oct  2 20:27:15 2018 (dataset_integrity.py:188)
[2018-10-03 20:27:02] [   ERROR] --- The dataset has been modified. The last_modified field does not match: old[Tue Oct  2 20:23:10 2018] new[Tue Oct  2 20:27:15 2018] (dataset_integrity.py:194)
[2018-10-03 20:27:02] [    INFO] --- Dataset loaded as images (setup.py:267)
[2018-10-03 20:27:02] [    INFO] --- Setting up model CNN_s1_2_2_4 (setup.py:86)
[2018-10-03 20:27:02] [    INFO] --- Loading weights for data balancing (setup.py:101)
[2018-10-03 20:27:02] [    INFO] --- Begin training (image_classification.py:73)
[2018-10-03 20:27:06] [    INFO] --- val   epoch[-1]: Acc@1=9.170	Loss=2.3108	Batch time=0.020 (0.002 to load data) (evaluate.py:151)
[2018-10-03 20:27:31] [    INFO] --- val   epoch[0]: Acc@1=45.090	Loss=1.5520	Batch time=0.020 (0.002 to load data) (evaluate.py:151)
[2018-10-03 20:27:59] [    INFO] --- val   epoch[1]: Acc@1=49.440	Loss=1.4108	Batch time=0.022 (0.002 to load data) (evaluate.py:151)
[2018-10-03 20:28:27] [    INFO] --- val   epoch[2]: Acc@1=52.060	Loss=1.3427	Batch time=0.024 (0.002 to load data) (evaluate.py:151)
[2018-10-03 20:28:56] [    INFO] --- val   epoch[3]: Acc@1=52.320	Loss=1.3316	Batch time=0.025 (0.002 to load data) (evaluate.py:151)
[2018-10-03 20:29:28] [    INFO] --- val   epoch[4]: Acc@1=54.880	Loss=1.2774	Batch time=0.026 (0.002 to load data) (evaluate.py:151)
[2018-10-03 20:29:33] [    INFO] --- test  epoch[4]: Acc@1=54.560	Loss=1.2713	Batch time=0.029 (0.002 to load data) (evaluate.py:151)
[2018-10-03 20:29:33] [    INFO] --- Training completed (image_classification.py:91)
[2018-10-03 20:29:33] [    INFO] --- Time taken for train/eval/test is: 0:02:32 (RunMe.py:257)
